### PR TITLE
Clean up the code and provide configuration templates

### PR DESCRIPTION
**Pull request by jonkiky**

### CODE REVIEW

File: src/bento/cartData.js [hunk 1]
```diff
@@ -30,17 +16,14 @@ export const myFilesPageData = {
   wizardIconSrc: 'https://raw.githubusercontent.com/CBIIT/bento-frontend/master/src/assets/icons/Cart-Wizard-Step3.svg',
   wizardIconAlt: 'Bento MyFiles Wizard',
   manifestFileName: 'BENTO File Manifest',
+  actionMessage: {
```
Comment (AjayDoddapaneni) [inline src/bento/cartData.js:19]: Looks like some change in the custodian file  is it necessary as we need to keep as similar as possible with BENTO?

Comment (jonkiky) [reply src/bento/cartData.js:19]: This is only a recommendation. You can ignore it if there is a better way.

Comment (AjayDoddapaneni) [reply src/bento/cartData.js:19]: I agree this is a better way than the previous one. Do we want to update the docs to represent this?

Comment (AjayDoddapaneni) [reply src/bento/cartData.js:19]: As we are the custodian for ICDC. I think. we can go ahead and merge this

**Review by AjayDoddapaneni — APPROVED** [review]

**Review by skiran86 — APPROVED** [review]

**Comment by skiran86** [comment]
@jonkiky conflicts

**Pull request merged**